packages/esm-patient-biometrics-app/src/clear-workspace.ts

```diff
@@ -0,0 +1,5 @@
+import { detachAll } from '@openmrs/esm-framework';
+
+export const clearWorkspace = (extensionSlotName) => {
```

Comment:
This currently just renames the `detachAll` function without doing anything else. -> I'd remove this entirely and just call `detachAll` directly (assuming that it's still called in the future which, considering @donaldkibet's comment, probably won't be the case).